Framework/Kernel/test/SmoothingTest.h

```diff
@@ -0,0 +1,111 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
```

Comment:
`#pragma once` isn't needed in the test header. They shouldn't be included into things other than a single test suite.